### DISCUSSION TITLE
Home Assistant PR#34748 : was that abandoned in favour of this?

### DISCUSSION THREAD

**Discussion by astubbs** [iloveicedgreentea/GoWatchIt#99] [General]
Hi there! Interesting project. I came across your PR: https://github.com/home-assistant/home-assistant.io/pull/34748. Was that abandoned in favour of this? What happened? It was frustrating to see home-assistant/home-assistant.io/pull/34748 closed with no close reason, and the PR conversation was locked to participants.

**Comment by iloveicedgreentea** [comment]
Other way around I wanted to add the core logic to HA but they rejected it in private convos so I got frustrated with the project and stopped contributing everything. I don’t agree with the logic of gate keeping integrations so I decided to rewrite this. I’m still toying with the idea of expanding the scope here to control devices and stuff so people who just have a theater wouldn’t need to use home assistant at all

I may contribute some changes as custom components like the plex integration but my appetite for contributing to home assistant isn’t very strong

**Reply by astubbs** [reply]
Gutted, sorry that happened. I just dipped my toe into Home Assistant today...
